### PR TITLE
added Windows Instructions for Visual Studio 15/17

### PR DESCRIPTION
**Pull request by germanafro**
This should help solving trouble integrating libfabric onto their Windows Platform.

As VS2017 Community Edition is free this solution is definitely available to anyone.

### CODE REVIEW

**Comment by shefty** [comment]
Can you move the windows text to the end of the readme, after the linux build data?  The assumption is that far more people care about this on linux than windows.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,46 @@ Libfabric distribution tarballs are already bootstrapped for you.
...
+
+  https://www.nuget.org/packages/NetworkDirect
+
+  or downloaded from:
+
+  https://www.microsoft.com/en-us/download/details.aspx?id=36043
+  on page press Download button and select NetworkDirect_DDK.zip.
+
+  Extract header files from downloaded
+  NetworkDirect_DDK.zip:`\NetDirect\include\` file into `<libfabricroot>/prov/netdir/NetDirect/`,
```
Comment (dmitrygladkov) [inline README.md]: `<libfabricroot>/prov/netdir/NetDirect/` -> `<libfabricroot>\prov\netdir\NetDirect\`
and please, update `/` -> `\` below

File: README.md [hunk 2]
```diff
@@ -37,6 +37,46 @@ Libfabric distribution tarballs are already bootstrapped for you.
...
+  By default the library will be compiled to `<libfabricroot>/x64/<yourconfigchoice>`
+
+- 3. linking your library
+  right click your project and select properties.
+  - VS 2017:
+    choose C/C++ > General and add `<libfabricroot>/include` to "Additional include Directories"
+    choose Linker > Input and add `<libfabricroot>/x64/<yourconfigchoice>/libfabric.lib` to "Additional Dependencies"
+    depending on what you are building you may also need to copy `libfabric.dll` into the targetfolder of your own project.
+
+  - VS 2015:
```
Comment (dmitrygladkov) [inline README.md]: VS 2015 and ICC are the same as for VS17.  Please, combine into single instruction

**Comment by shefty** [comment]
Note that I will manually squash the 2 commits into one when merging.

**Pull request merged**